gsa/src/web/pages/results/filterdialog.js

```diff
@@ -112,10 +126,20 @@ const ResultsFilterDialogComponent = ({
       onChange={onFilterValueChange}
     />
 
+    <OwnerGroup filter={filter} onChange={onSearchTermChange} />
+
+    <VulnerabilityGroup filter={filter} onChange={onSearchTermChange} />
+
+    <HostGroup filter={filter} onChange={onSearchTermChange} />
+
+    <LocationGroup filter={filter} onChange={onSearchTermChange} />
+
     <FirstResultGroup filter={filter} onChange={onFilterValueChange} />
 
     <ResultsPerPageGroup filter={filter} onChange={onFilterValueChange} />
 
+    <CvssBaseGroup filter={filter} onChange={onFilterValueChange} />
```

Comment:
Can we move the cvssbasgroup below the severity class?

Reply:
Will do :)